### PR TITLE
[SYTO-1104] FE: Update Video Modal z-index in Mortar

### PR DESCRIPTION
**Pull request by bpearsoncse**
https://jira.natgeo.com/browse/SYTO-1104

### CODE REVIEW

File: src/modules/modals/_styles.scss [hunk 1]
```diff
@@ -23,7 +23,7 @@
   top: 7.5vh;
   visibility: hidden;
   width: 85vw;
-  z-index: $mt3_z-top;
+  z-index: 2147483647; // this is where the universe ends.
```
Comment (pgurley) [inline src/modules/modals/_styles.scss]: I feel like this should be a variable in styles/themes/default/vars/_z.scss

Comment (bpearsoncse) [inline src/modules/modals/_styles.scss]: I am going to go ahead and make multiple variables with the ranges as comments based on IAB guidelines then.

**Comment by bpearsoncse** [comment]
Changes made - back to you @pgurley

File: src/styles/themes/default/vars/_z.scss [hunk 2]
```diff
@@ -1,5 +1,64 @@
...
+* Flyout menus, video players, carousels etc.
+* Can be range from 3000 - 3999.
+**/
+$mt3_z-js-menus-widgets: 3000;
+
+/**
+* Main navigation menus and registration package.
+* Can be range from 4000 - 4999.
+**/
+$mt3_z-global-sticky-nav: 4000;
```
Comment (pmourelle) [inline src/styles/themes/default/vars/_z.scss]: Not sure about naming, remember mortar should be agnostic about where is applied.

File: src/modules/modals/Modal.js [hunk 3]
```diff
@@ -12,13 +12,28 @@ class Modal extends Component {
...
+  componentDidMount() {
+    document.body.addEventListener('keydown', this.handleKeyDown);
+  }
+
+  componentWillUnmount() {
+    document.body.removeEventListener('keydown', this.handleKeyDown);
+  }
+
+  handleKeyDown(event) {
+    if (event.keyCode == 27 /*esc closes modal*/) {
```
Comment (pmourelle) [inline src/modules/modals/Modal.js]: Maybe commenting out ot the guard? I certainly doubt any lint passes this.

File: src/styles/themes/default/vars/_z.scss (hunk 2 above)
Comment (bpearsoncse) [inline src/styles/themes/default/vars/_z.scss]: renamed to z-nav-menus

**Comment by bpearsoncse** [comment]
Changes made back to you @pgurley

**Comment by pgurley** [comment]
Merged

**Pull request merged**